examples/AspNetCoreMvc/Properties/launchSettings.json

```diff
@@ -55,7 +55,13 @@
         "OTEL_DOTNET_AUTO_TRACES_CONSOLE_EXPORTER_ENABLED": "true",
         "OTEL_DOTNET_AUTO_TRACES_PLUGINS": "Examples.AspNetCoreMvc.OtelSdkPlugin, Examples.AspNetCoreMvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null",
         "OTEL_SERVICE_NAME": "StartupHook.Self-hosted",
-        "OTEL_TRACES_EXPORTER": "otlp"
+        "OTEL_TRACES_EXPORTER": "otlp",
+        "OTEL_DOTNET_AUTO_HOME": "$(SolutionDir)bin\\tracer-home",
+        "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "OpenTelemetry.AutoInstrumentation.AspNetCoreBootstrapper",
+        "OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED": "true",
+        "OTEL_LOGS_EXPORTER": "otlp",
```

Comment:
As for the traces exporter this should be the default. In order to keep the launching profiles small I think it is best to remove any env var setting the same as their default value.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1135